backend/docs/api/management_v2_deployments.yaml

```
@@ -571,6 +571,105 @@ paths:
         Lists all release update types.
       tags:
       - Management API
+  /artifacts/list:
```

Can we make the `artifacts` RESTful again? :upside_down_face:
```suggestion
  /artifacts:
```

---

+1